ros/lib/utils.py

```diff
@@ -111,8 +112,10 @@ def sort_io_dict(performance_utilization: dict):
     return performance_utilization
 
 
-def system_ids_by_account(account_number):
+def system_ids_by_account(account_number, fetch_records=False):
```

Comment:
@kgaikwad IMO we should leverage function overloading as long it does not get too complex, this can keep things simple without adding extra lines of code.